CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+## Hello Contributors!
+  Thx for your interest! We're so glad you're here. 
+
+### Important Resources
+  - bugs: [https://github.com/tinkerbell/osie/issues](https://github.com/tinkerbell/osie/issues)
```

Comment:
Why not make this a `####`+ heading? This way we'd get an anchor to link to more easily.

#### [Bugs](https://github.com/tinkerbell/osie/issues)

Also issues isn't just for bugs, but also feature requests too right?

Reply:
If you'd like feature requests done via 'issues', yes, definitely. This is a minimum request. Please expand as needed.

The anchor link can be different, too; definitely.